e2e/cypress/integration/account_settings/theme_color/custom_colors_sidebar_styles_spec.js

```diff
@@ -147,3 +147,84 @@ describe('AS14318 Theme Colors - Color Picker', () => {
         });
     });
 });
+
+describe('MM-14318 Theme Colors', () => {
+    beforeEach(() => {
+        //Login and navigating to settings
+        cy.apiLogin('user-1');
+        cy.toAccountSettingsModal(null, true);
+        cy.get('#displayButton').click();
+        cy.get('#themeTitle').click();
+        cy.get('#customThemes').click();
+    });
+
+    after(() => {
+        cy.defaultTheme();
+    });
+
+    it('Should be able to use color picker input and change Sidebar theme color', () => {
+        cy.get('#sidebarStyles').click();
+
+        // # Click the Sidebar BG setting
+        cy.get('[data-test-id=sidebarBg-squareColorIcon]:visible').first().click();
```

Comment:
If `id` is implemented, then this can be,
```suggestion
        cy.get('#sidebarBgSquareColorIcon').click();
```

Same for other instances.

Reply:
Yes, I agree that the tests are easier to read using ids, I just update to use ids.

Reply:
For the future, we can also use `cy.getByTestId('foobarbaz')`.